hazelcast/src/main/java/com/hazelcast/internal/auditlog/impl/AuditlogTypeIds.java

```diff
@@ -18,7 +18,69 @@
 
 public final class AuditlogTypeIds {
 
-    public static final String CONNECTION_ASKS_PROTOCOL = "HZ-1001";
+    // Network Events
+    /**
+     * Event type ID: Connection accepted.
+     */
+    public static final String NETWORK_CONNECT = "HZ-0101";
```

Comment:
Can we move these into a public package? I guess these are part of public API now, given that we can't change the IDs after we've released them. Also, maybe we can refer to this file from the `AuditlogService` javadoc.

Reply:
Good point. I'll move the class and update the JavaDoc.